lisa/sut_orchestrator/azure/tools.py

```diff
@@ -55,3 +57,53 @@ def get_generation(self) -> str:
         else:
             generation = "1"
         return generation
+
+
+class LisDriver(Tool):
+    """
+    This is a virtual tool to detect/install LIS drivers.
+    """
+
+    __version_pattern = re.compile(r"^version:[ \t]*([^ \n]*)")
+
+    @property
+    def command(self) -> str:
+        return "modinfo hv_vmbus"
+
+    def _check_exists(self) -> bool:
+        if not self.node.os.is_posix:
+            return False
+
+        if isinstance(self.node.os, Fedora):
```

Comment:
Nitpick: bail out if not Fedora, less indentation. Also, return those conditions directly, as they are a bool value already.